api/mapping/services_rules.py

```diff
@@ -59,12 +59,18 @@ def get_omop_field(destination_field,
     Returns:
       - OmopField : the destination field object
     """
+
     #if we haven't specified the table name
     if destination_table == None:
         #look up the field from the "allowed_tables"
         omop_field = OmopField.objects\
-                               .filter(table__table__in=m_allowed_tables)\
-                               .get(field=destination_field)
+                              .filter(field=destination_field)
```

Comment:
this was previously crashing when None was retrieved due to the table not being in m_allowed_tables